tests/meson.build

```diff
@@ -1,6 +1,11 @@
-gtest_proj = subproject('gtest')
-gtest_dep = gtest_proj.get_variable('gtest_dep')
-gmock_dep = gtest_proj.get_variable('gmock_dep')
+gtest_dep = dependency('gtest', required : false)
```

Comment:
As far as I can see this can be done much more concisely with just
```meson
gtest_dep = dependency('gtest', fallback: ['gtest', 'gtest_dep'])
gmock_dep = dependency('gmock', fallback: ['gmock', 'gmock_dep'])
```

Also, when allowing a system package, some minimum version should be specified. `>=1.14.0` should be fine, since that's what TypesettingTools/Aegisub has been using for a while now.

Reply:
Thank you, applied, and I only tested the system dependency conditional branch as before.